docs/docs/user-guides/trustless-faq.md

```diff
@@ -1,4 +1,4 @@
-# FAQ
+# Trustless FAQ
```

Comment:
Are these FAQ's solely for app.trustless.zone?

Reply:
yes, this one is for trestles only. we plan on adding more FAQs later